File: RUBY/Cap07/c07ex03.rb
# c07ex03.rb

#!/usr/bin/env ruby

def troca
  x = $a
  $a = $b
  $b = x
end

print "Entre um valor para a variavel <A>: "
$a = gets.chomp.to_i

print "Entre um valor para a variavel <B>: "
$b = gets.chomp.to_i

troca()

puts "\nVariavel <A> com valor #{$a}"
puts "Variavel <B> com valor #{$b}"

print "\nTecle <Enter> para encerrar... "
gets

File: LUA/Cap07/c07ex03.lua
-- c07ex03.lua

function troca()
  local x = a
  a = b
  b = x
end

io.write("Entre um valor para a variavel <A>: ")
a = tonumber(io.read())

io.write("Entre um valor para a variavel <B>: ")
b = tonumber(io.read())

troca()

print("\nVariavel <A> com valor " .. a)
print("Variavel <B> com valor " .. b)

io.write("\nTecle <Enter> para encerrar... ")
io.read()
